From a979c8d360bd2d930a51766a2bc42ede3a0a1ac6 Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Thu, 10 Aug 2017 08:57:05 +0300
Subject: [PATCH 08/19] config: Allow gms to access persistent data partition

This fixes nonfunctioning factory reset protection and addresses
numerous PersistentDataBlockService errors during boot up.

Change-Id: Ifb24b2cffc814a3c2f61eda619f2a5cbe173d0f8
---
 core/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index 467391172b..825d6b67b3 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -3253,7 +3253,7 @@
     </string-array>
 
     <!-- Flag indicating which package name can access the persistent data partition -->
-    <string name="config_persistentDataPackageName" translatable="false"></string>
+    <string name="config_persistentDataPackageName" translatable="false">com.google.android.gms</string>
 
     <!-- Flag indicating which package name can access DeviceConfig table -->
     <string name="config_deviceConfiguratorPackageName" translatable="false"></string>
-- 
2.34.1

